vaadin-platform-react-hybrid-test/src/main/frontend/views/hilla/components.tsx

```diff
@@ -0,0 +1,315 @@
+import {
+    Accordion,
+    AccordionHeading,
+    AccordionPanel,
+    Avatar,
+    AvatarGroup,
+    Button,
+    Checkbox,
+    CheckboxGroup,
+    ComboBox,
+    ConfirmDialog,
+    ContextMenu,
+    DatePicker,
+    DateTimePicker,
+    Details,
+    DetailsSummary,
+    Dialog,
+    DrawerToggle,
+    EmailField,
+    FormLayout,
+    Grid,
+    GridColumn,
+    GridSelectionColumn,
+    GridSortColumn,
+    GridSorter,
+    GridTreeToggle,
+    HorizontalLayout,
+    Icon,
+    Iconset,
+    IntegerField,
+    Item,
+    ListBox,
+    LoginForm,
+    LoginOverlay,
+    MenuBar,
+    Message,
+    MessageInput,
+    MessageList,
+    MultiSelectComboBox,
+    Notification,
+    NumberField,
+    PasswordField,
+    Popover,
+    ProgressBar, RadioButton, RadioGroup,
+    Scroller,
+    Select, SideNav, SideNavItem, SplitLayout, Tab,
+    Tabs, TabSheet, TabSheetTab,
+    TextArea,
+    TextField,
+    TimePicker,
+    Tooltip,
+    Upload,
+    VerticalLayout,
+    VirtualList,
+    VirtualListItemModel
+} from "@vaadin/react-components";
+import type { ViewConfig } from "@vaadin/hilla-file-router/types.js";
+import {
+    Board,
+    BoardRow,
+    Chart,
+    ChartSeries,
+    CookieConsent,
```

Comment:
there is a new component called, dash board in V24.6.. 
should we add it here?

Reply:
We could, This just does the same test as lit hybrid that there are no exceptions thrown....